compiler-cli/src/compile_package.rs

```diff
@@ -47,6 +47,7 @@ pub fn command(options: CompilePackage) -> Result<()> {
         &target,
         ids,
         ProjectIO::new(),
+        todo!("I'm not sure what should go here"),
```

Comment:
@lpil I'm not sure what to do here, in the compiler-core package I had access to a `is_root` variable but here there's no such thing